aks/postgres/resources.tf

```diff
@@ -7,6 +7,8 @@ locals {
   azure_private_endpoint_name = "${var.azure_resource_prefix}-${var.service_short}-${var.config_short}-pg${local.name_suffix}-pe"
 
   kubernetes_name = "${var.service_name}-${var.environment}-postgres${local.name_suffix}"
+
+  azure_backup_storage = var.use_azure && var.azure_backup_storage_account
```

Comment:
Can we name it clearly like a boolean? Like `enable_azure_backup_storage`, `create_azure_backup_storage`... ?

Reply:
Yeah no problem, I've also renamed the input variable for the entire module to match, are we okay with making that change here?